src/content/docs/frameworks/Asp.netCore/AspNetCore-case-study.md

```diff
@@ -0,0 +1,316 @@
+- **2016**: Release of ASP.NET Core 1.0, a major overhaul introducing a cross-platform, unified framework for web development.
+- **2017**: ASP.NET Core 2.0 debuts, enhancing compatibility with .NET Framework and .NET Standard, simplifying the transition for existing .NET developers.
+- **2018**: ASP.NET Core 2.1 introduces SignalR, Razor Class Libraries, and Identity UI; followed by 2.2, which adds endpoint routing and health checks.
+- **2019**: ASP.NET Core 3.0 brings support for C# 8, Windows desktop apps, and performance improvements, marking a significant update.
+- **2020**: Launch of .NET 5.0, transitioning to a unified .NET platform, streamlining the naming and further integrating ASP.NET.
+- **2021**: .NET 6.0, an LTS release, introduces minimal APIs and other enhancements, focusing on performance and productivity.
+- **2022**: .NET 7 continues to build on .NET 6’s foundations, emphasizing refinements and expanding features.
+- **2023**: .NET 8 released
+
+## Main Features
+
+Asp.net core provides a opinionated framework for creating web applications, it has built in security and helps developers follow best practices by making it easier to seperate conserns. which may help keep code decoupled, easier to test and change. It also comes with some built in functionality that improves the developer experience and code quality. 
+
+- **Model binding:** automates the process of converting HTTP requests into .NET objects. It maps data from HTTP requests (including query strings, form values, route data, and JSON bodies) to action method parameters. Making it so you can use the properties you expect as parameters in your controllers, instead of manually having to extract them from the request. 
+  
+- It also provides model validation, making it simple to validate the request, and provide quick error messages if needed.     
+- **Supports dependency injection:** Provides an IOC container that you can register services with, and use to inject your dependency at runtime. Abstractions make it so you only need to register the service and its life time. Then you can add the service either by class constructor or method injection.  
+- **Natively supports the MVC pattern:** Asp.net Core provides an opinionated way of using the mvc pattern, having predefined interfaces and classes to implement or inherit from to streamline the process. 
+- **Attribute routing:** You can create your routes directly on your action methods inside your  controllers making development process faster, and gives you an better overview of what action methods are coupled with what endpoints.
+
+
+## Market Comparison
+
+Instead of having and extensive comparison how asp.net core compares to many of the popular frameworks like Laravel, Spring, Django and Express.js. I have decided to solely compare it to express.js since i expect that to be the most likely backend framework front end developers are familiar with. 
```

Comment:
Fix the tone to the third person. Consider this suggestion
```suggestion
Instead of providing an extensive comparison of how ASP.NET Core compares to many popular frameworks like Laravel, Spring, Django, and Express.js, the decision has been made to solely compare it to Express.js. This choice is based on the expectation that Express.js is the backend framework most likely to be familiar to frontend developers.